config/dynamic.c

```diff
@@ -450,12 +455,13 @@ set_dyc_timeout(unsigned int sec,
 }
 
 int
-run_dynamic_config(const char *server_path, struct dynamic_config *dy_conf)
+run_dynamic_config(struct gk_config *gk_conf,
+	const char *server_path, struct dynamic_config *dy_conf)
 {
 	int ret;
 	struct sockaddr_un server_addr;
 
-	if (server_path == NULL || dy_conf == NULL) {
+	if (server_path == NULL || gk_conf == NULL || dy_conf == NULL) {
```

Comment:
Good catch. Actually, I delayed to deal with it to when GT comes, since we haven't integrated it yet. I will add a comment here.